jooby/src/test/java/org/jooby/internal/RoutePatternTest.java

```diff
@@ -139,6 +139,72 @@ public void any2() {
   }
 
   @Test
+  public void anyNamed() {
+    new RoutePathAssert("GET", "com/**:rest")
+        .matches("GET/com/test.jsp", vars -> assertEquals("test.jsp", vars.get("rest")))
+        .matches("GET/com/a/test.jsp", vars -> assertEquals("a/test.jsp", vars.get("rest")))
+        .matches("GET/com/", vars -> assertEquals("", vars.get("rest")))
+        .butNot("GET/test.jsp");
+  }
+
+  @Test
+  public void anyNamedSpring() {
+    new RoutePathAssert("GET", "com/{rest:**}")
+        .matches("GET/com/test.jsp", vars -> assertEquals("test.jsp", vars.get("rest")))
+        .matches("GET/com/a/test.jsp", vars -> assertEquals("a/test.jsp", vars.get("rest")))
+        .matches("GET/com/", vars -> assertEquals("", vars.get("rest")))
+        .butNot("GET/test.jsp");
+  }
+
+  @Test
+  public void anyNamedInner() {
+    new RoutePathAssert("GET", "com/**:rest/bar")
+        .matches("GET/com/foo/bar", (vars) -> assertEquals("foo", vars.get("rest")))
+        .matches("GET/com/a/foo/bar", (vars) -> assertEquals("a/foo", vars.get("rest")))
+        .butNot("GET/com/foo/baz")
+        .butNot("GET/test.jsp");
```

Comment:
Is it worth testing `matches("GET/com/foo/bars/bar", (vars) -> assertEquals("foo/bars", vars.get("rest")))`? Or `matches("GET/com/foo/abar/bar", (vars) -> assertEquals("foo/bars", vars.get("rest")))`
Again, it seems to me another corner case but not 100% sure.